webviz_subsurface/plugins/_parameter_analysis/models/parameters_model.py

```diff
@@ -206,39 +206,94 @@ def make_grouped_plot(
                         ),
                         20,
                     ),
+                    facet_row_spacing=max((0.08 - (0.00071 * len(parameters))), 0.03),
                     color="ENSEMBLE",
                     color_discrete_sequence=self.colorway,
+                    custom_data=["PARAMETER"],
                 )
                 .update_xaxes(
                     matches=None,
                     fixedrange=True,
                     title=None,
-                    showticklabels=(len(parameters) < 20),
+                    showticklabels=len(parameters) <= 100,
+                    tickangle=0,
+                    tickfont_size=max((18 - (0.4 * len(parameters))), 8),
                 )
+                .update_yaxes(showticklabels=False)
                 .for_each_trace(
                     lambda t: t.update(
                         hoveron="violins",
-                        hoverinfo="name",
+                        hoverinfo="none",
                         meanline_visible=True,
                         orientation="h",
                         side="positive",
-                        width=3,
+                        width=2,
                         points=False,
                     )
                 )
                 .for_each_annotation(
                     lambda a: a.update(
-                        hovertext=a.text.split("=")[-1],
-                        text=(a.text.split("=")[-1]) if len(parameters) < 40 else "",
+                        text=(a.text.split("=")[-1]),
+                        visible=len(parameters) <= 42,
+                        font_size=max((18 - (0.4 * len(parameters))), 8),
```

Comment:
Same question on scaling for different screens/windows sizes. 

Also: expression for font_size is the same as above for tickfont_size. Could consider to create a small function to share the logic.

Reply:
I agree with the small function, perhaps this should be part of a generalized utility library?

Reply:
Maybe something similar, could be soemthing we can consider at some point at least 😊 Not sure how well the equation would fit for a general plugin 🤔